hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveSchemaUtil.java

```diff
@@ -181,16 +182,16 @@ public void testComplexTypeAndTypeInfoConvert() {
 
   protected Schema getSchemaWithSupportedTypes() {
     return new Schema(
-        optional(0, "c_float", Types.FloatType.get()),
-        optional(1, "c_double", Types.DoubleType.get()),
-        optional(2, "c_boolean", Types.BooleanType.get()),
-        optional(3, "c_int", Types.IntegerType.get()),
-        optional(4, "c_long", Types.LongType.get()),
-        optional(5, "c_binary", Types.BinaryType.get()),
-        optional(6, "c_string", Types.StringType.get()),
-        optional(7, "c_timestamp", Types.TimestampType.withoutZone()),
-        optional(8, "c_date", Types.DateType.get()),
-        optional(9, "c_decimal", Types.DecimalType.of(38, 10)));
+        optional(0, "c_float", Types.FloatType.get(), ""),
+        optional(1, "c_double", Types.DoubleType.get(), ""),
+        optional(2, "c_boolean", Types.BooleanType.get(), ""),
+        optional(3, "c_int", Types.IntegerType.get(), ""),
+        optional(4, "c_long", Types.LongType.get(), ""),
+        optional(5, "c_binary", Types.BinaryType.get(), ""),
+        optional(6, "c_string", Types.StringType.get(), ""),
+        optional(7, "c_timestamp", Types.TimestampType.withoutZone(), ""),
+        optional(8, "c_date", Types.DateType.get(), ""),
+        optional(9, "c_decimal", Types.DecimalType.of(38, 10), ""));
```

Comment:
Just to make sure: this should not use empty string if there is no comment. If there is no comment, then it should use null. It's a little misleading to see empty string like this in tests. Could you remove half of them and add non-empty doc strings to the other half?

Reply:
Fixed.